### PR TITLE
[webviz]Remove rosbag base url to adapt new minio

### PR DESCRIPTION
**Pull request by surajhpatil**
Following things have changed:

- Removed `enway_config` which contained `remoteBagConnectionURL`.
 We do not need this anymore. As we get complete URL from `minio` api with which webviz can work.
- Added new util method to sanitize the rosmessages being sent for parsing
 Current `rosbag` package fails to parse some standard rosmessages with bool as datatype.
 Check this [issue](https://github.com/cruise-automation/rosbag.js/issues/56) for more.
 The `getSanitizeMessageDefinition()` function should handle these edge cases.
- Npm audit fixes

### CODE REVIEW

File: packages/webviz-core/src/components/PlayerManager.js [hunk 1]
```diff
@@ -66,12 +65,17 @@ function PlayerManager({ importPanelLayout, children, userNodes, setNodeDiagnost
   React.useEffect(
     () => {
       const params = new URLSearchParams(window.location.search);
-      console.log(remoteBagConnectionURL);
-      const remoteBagBaseURL = remoteBagConnectionURL;
-      const remoteDemoBagUrl = "https://open-source-webviz-ui.s3.amazonaws.com/demo.bag";
+      const remoteDemoBagUrl =
+        "http://storage.enway.ai:9000/bags-studio-16/studio_16_01.bag?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20191126%2F%2Fs3%2Faws4_request&X-Amz-Date=20191126T123246Z&X-Amz-Expires=432000&X-Amz-SignedHeaders=host&X-Amz-Signature=336681d16d39b0100893a6eaf26c7b3755feedbe5110f8335810cde27511eb16";
```
Comment (stefan-enway) [inline packages/webviz-core/src/components/PlayerManager.js]: we intend to only use https:// for storage.enway.ai

Comment (stefan-enway) [reply packages/webviz-core/src/components/PlayerManager.js]: can you try if https:// works out please?

**Review by stefan-enway — CHANGES_REQUESTED** [review]
see comments

File: packages/webviz-core/src/components/PlayerManager.js (hunk 1 above)
Comment (surajhpatil) [reply packages/webviz-core/src/components/PlayerManager.js]: okay, by default it gives `http`.
Will try with `https`

Comment (surajhpatil) [reply packages/webviz-core/src/components/PlayerManager.js]: `https` doesn't work as of now it seems.
```
This site can’t provide a secure connectionstorage.enway.ai sent an invalid response.
ERR_SSL_PROTOCOL_ERROR
```
Is what it says in the browser.

**Comment by surajhpatil** [comment]
> see comments

Updated url to `https` regardless the current state of `https`.
As that's what we should use anyways.

**Comment by surajhpatil** [comment]
BTW, we will also need  `https` for the [rainbow](http://rainbow.local.enway.ai:8080/)
Current it is also on `http` and `https` doesn't work there.
I will also update that URL ([for  another PR](https://github.com/enwaytech/cleansquare_ros/pull/1994)) to `https`

File: packages/webviz-core/src/components/PlayerManager.js [hunk 2]
```diff
@@ -66,12 +65,17 @@ function PlayerManager({ importPanelLayout, children, userNodes, setNodeDiagnost
   React.useEffect(
     () => {
       const params = new URLSearchParams(window.location.search);
-      console.log(remoteBagConnectionURL);
-      const remoteBagBaseURL = remoteBagConnectionURL;
-      const remoteDemoBagUrl = "https://open-source-webviz-ui.s3.amazonaws.com/demo.bag";
+      const remoteDemoBagUrl =
+        "https://storage.enway.ai:9000/bags-studio-16/studio_16_01.bag?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20191126%2F%2Fs3%2Faws4_request&X-Amz-Date=20191126T123246Z&X-Amz-Expires=432000&X-Amz-SignedHeaders=host&X-Amz-Signature=336681d16d39b0100893a6eaf26c7b3755feedbe5110f8335810cde27511eb16";
```
Comment (stefan-enway) [inline packages/webviz-core/src/components/PlayerManager.js]: https lives at port 443 and has valid Let's Encrypt certificates

Comment (stefan-enway) [reply packages/webviz-core/src/components/PlayerManager.js]: need to add storage.enway.ai to alt_names

Comment (surajhpatil) [reply packages/webviz-core/src/components/PlayerManager.js]: done! updated to new url.

**Review by stefan-enway — APPROVED** [review]
LGTM

**Pull request merged**